schemas.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Validation schemas for incoming requests
+*/
+
+var Joi = require('joi');
+var mimes = Object.keys(require('mime-db'));
+
+exports.file = Joi.object().keys({
+  type: Joi.string().valid(mimes),
```

Comment:
IMO that would be an enhancement. let's not worry about that for now, ship a simple version of this and move the thread to the issues.